File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int l = 1;
        int r = 1;

        int n = nums.size();

        while(r < n){
            if(nums[r] != nums[r-1]){
                nums[l] = nums[r];
                l+=1;
            }
            r+=1;
        }

        return l;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:

        seen = {}
        seen_index = None

        for i, ele in enumerate(nums):
            to_find = target - ele
            if to_find in seen:
                return [seen[to_find], i]

            seen[ele] = i

        return 0, 0
            

        